tools/download_models.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+model_dir=$ONNX_HOME/.onnx/models
```

Comment:
How is `ONNX_HOME` set? I dont see it set in the Dockerfile. Also is it possible to set this to a default(ie `ONNX_HOME=$HOME`) when it wasn't set by the user?

Reply:
The env var ONNX_HOME is used in the onnx unit test. It indicates where the test models and data are stored. I think we can set to to whatever value we want. I will change to set it to home.

Reply:
But this script is called from the Dockerfile not onnx test suite. Can the script fallback to `$HOME` if `ONNX_HOME` is not defined?

Reply:
Done